app/lint.xml

```diff
@@ -14,6 +15,7 @@
     <issue id="MissingTranslation" severity="ignore" />
     <issue id="PluralsCandidate" severity="ignore" />
     <issue id="StringFormatCount" severity="ignore" />
+    <issue id="StringFormatInvalid" severity="warning" />
```

Comment:
I think we should add comments explaining why we're ignore/warning certain checks.

Reply:
Oh, I see the "Lints that don't apply to our translation process" comment now.

Is StringFormatInvalid a bug that could lead to crashing? I feel like maybe this should be "Error" (though I think that'd break the current l10n process...).

Reply:
It seemed to hit strings where the translations didn't use the placeholder. That has come up in the past and afaik we allowed that for certain translations. But yeah, we should double check if that crashes the app.